tools/shared.py

```diff
@@ -367,11 +369,17 @@ def parse_config_file():
     'PORTS',
   )
 
+  # A subset of the above keys that represent commands. In this case its useful to
+  # use shlex.split on the environment variable.
+  COMMAND_KEYS = ('NODE_JS', 'JAVA', 'WASMER', 'WASMTIME', 'JS_ENGINE', 'CLOSURE_COMPILER')
```

Comment:
this change seems slightly risky and maybe might be worth splitting out?

Reply:
Use you are right there is a slight risk.  The user who would could be broken by this would be somebody who does `EM_NODE_JS='/path/with spaces/bin/node'` because now that would be interpreted as two different args.

I can split this out and add a test for it.   I don't think it should be too risky since I don't imagine there are many users of EM_XXX environment variables who also want to include pathnames with spaces (normally thats mostly windows users).